front/lib/resources/content_fragment_resource.ts

```diff
@@ -58,6 +58,18 @@ export class ContentFragmentResource extends BaseResource<ContentFragmentModel>
     );
   }
 
+  static async fromMessageId(id: ModelId) {
```

Comment:
Ideally this should live in the future `MessageResource` 🙃.

Reply:
But not yet right? :upside_down_face:

Reply:
Yes not yet.